docker/Dockerfile

```diff
@@ -52,7 +45,6 @@ EXPOSE 3141
 
 # sort out dynamic user login
   COPY        docker/entrypoint.sh.addHostUserToContainer /usr/local/bin/entrypoint.sh.addHostUserToContainer
-  RUN         chmod +x /usr/local/bin/entrypoint.sh.addHostUserToContainer
```

Comment:
Yes, it shows that in the diff:
```
old mode 100644
new mode 100755
```
Or in the git UI 
![image](https://cloud.githubusercontent.com/assets/6473438/25030420/e51a6b62-2092-11e7-8062-effcdf7c6451.png)